kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/YearEndGeneralErrorCorrectionForm.java

```diff
@@ -0,0 +1,14 @@
+package edu.arizona.kfs.fp.document.web.struts;
+
+public class YearEndGeneralErrorCorrectionForm extends GeneralErrorCorrectionForm {
+
+	public YearEndGeneralErrorCorrectionForm() {
+		super();
+	}
+	
+	@Override
+	protected String getDefaultDocumentTypeName() {
+		return "YEGE";
```

Comment:
KFSConstants.FinancialDocumentTypeCodes is deprecated.  I do not see any other constants that are not deprecated.  Also, the baseline version uses the same "magic string". Should this be left as is given the deprecated constant?